.github/workflows/wheels.yml

```diff
@@ -0,0 +1,54 @@
+name: Build wheels
+
+on: [push]
+# workflow_dispatch:{}
+
+env:
+  DEBIAN_FRONTEND: noninteractive
+  FORCE_CUDA: 1
+  # all values:
+  # 3.5;3.7;5.0;5.2;5.3;6.0;6.1;6.2;7.0;7.2;7.5;8.0;8.6
+  # with optional +PTX
+  # https://github.com/pytorch/vision/blob/0610b13ac4af3717f538454a9c6b1f441cb386f3/packaging/pkg_helpers.bash#L63
+  # you need at least cuda 5.0 for some of the stuff compiled here.
+  TORCH_CUDA_ARCH_LIST: "6.0 6.1 7.0 7.5 8.6"
+
+jobs:
+  build_wheels:
+    strategy:
+      matrix:
+        # python: ["3.7", "3.8", "3.9", "3.10"]
+        python: ["3.10"]
+         # according to the available pytorch cuda versions
+        # cuda: ["cu113", "cu116"]
+        cuda: ["cu116"]
+    name: py${{ matrix.python }}-${{ matrix.cuda }}
+    runs-on: ubuntu-latest
+    timeout-minutes: 360
+    container:
+      image: python:${{ matrix.python }}-slim
+    steps:
+      - name: Install dependencies
+        run: apt-get update && apt-get install gpg wget git software-properties-common rename -y
+
+      - name: Install nvcc
+        run: >
+          wget https://developer.download.nvidia.com/compute/cuda/repos/debian11/x86_64/cuda-keyring_1.0-1_all.deb &&
+          dpkg -i cuda-keyring_1.0-1_all.deb &&
+          apt-get update &&
+          apt-get install cuda-nvcc-11-8 cuda-libraries-dev-11-8 -y
```

Comment:
we should install the version corresponding to `matrix.cuda`. There are some available github-action for that to help apparently: https://github.com/Jimver/cuda-toolkit

Reply:
I am going to have to try this first.

Currenty, the build happens in a debian slim container, which should (at least in theory) work with this action.

PyTorch logs a warning to the console if the nvcc version is not the same as the one it was compiled with, but because it is only a minor version, pytorch logs that "this is likely not a problem" so I just winged it.

Reply:
I ended up not using this action since it only works on debian but the container I am using is centos